packages/itwinui-css/backstop/tests/input.html

```diff
@@ -502,5 +505,118 @@ <h2>No wrapping label</h2>
       </div>
 
     </section>
+
+    <br />
+
+    <h2>Search box</h2>
+    <section id="demo-search-box">
+
+      <h3>Current method</h3>
+      <div class="iui-input-container iui-inline-icon">
+        <input
+          placeholder="Placeholder…"
+          type="search"
+          class="iui-input"
+          id="test-1"
+        />
+        <svg-search
+          class="iui-input-icon"
+          aria-hidden="true"
+        />
+      </div>
+
+      <h3>Flex box</h3>
+      <div
+        class="iui-input-flex-container"
+        data-iui-size="small"
+      >
+        <span>
+          <svg-search
+            class="iui-input-icon"
+            aria-hidden="true"
+          /></span>
+        <input
+          placeholder="Placeholder…"
+          type="search"
+          class="iui-invisible-borders"
+          id="test-1"
+          value="test search input"
+        />
+        <button
+          class="iui-button iui-end-icon"
```

Comment:
how will `iui-end-icon` be applied in react?

Reply:
Removed from html as it is not needed. 
Researching how would work without <Icon> and with other buttons.

Reply:
Not relevant because I've created SearchBox.Icon component for users to use.